breaking_changes.md

```diff
@@ -0,0 +1,40 @@
+# Breaking changes
+
+
+The following table describes potential changes and if they require a version bump.
+
+Keep in mind there may be instances where these rules are violated if there is a pressing performance requirement from the monolith. We will try to keep violations to an absolute minimum, and broadcast the changes in advance if possible.
+
+For the purposes of this table, parameter means input parameter, whereas attribute refers to the response object.
+
+
+Change Description                                 |  Version Bump Required
+---------------------------------------------------|-----------------------
+Update description/summary/example                 | &#10007;
+Adding attribute or parameter                      | &#10007;
+Removing optional attribute                        | &#10007;
+Removing required attribute                        | &#10003;
+Removing parameter                                 | &#10007;
+Optional attribute becomes required                | &#10007;
+Required attribute becomes optional                | &#10003;
+Optional parameter becomes required                | &#10003;
+Required parameter becomes optional                | &#10007;
+Changing attribute or parameter name               | &#10003;
+Changing attribute or parameter type<sup>1</sup>   | &#10003;
+Changing attribute format<sup>1</sup>              | &#10007;
+Changing parameter format<sup>1</sup>              | &#10003;
```

Comment:
If we narrow the scope of a parameter (e.g. a param that used to accept int64's now only accepts int32's) then when we deploy the change, anyone using the wider format will start getting error limited. I would argue this should be a version bump if it is a narrowing update.

Reply:
I agree, maybe I should list each transition individually though. narrowing isn't that specific. 

will update this part